docs/content/docs/dev/configuration/overview.md

```diff
@@ -0,0 +1,219 @@
+
+### Maven command
+```bash
+$ mvn archetype:generate                \
+  -DarchetypeGroupId=org.apache.flink   \
+  -DarchetypeArtifactId=flink-quickstart-java \
+  -DarchetypeVersion={{< version >}}
+```
+This allows you to name your newly created project and will interactively ask you for the groupId,
+artifactId, and package name.
+
+### Quickstart script
+```bash
+$ curl https://flink.apache.org/q/quickstart.sh | bash -s {{< version >}}
+```
+
+{{< /tab >}}
+{{< tab "Gradle" >}}
+You can create a project with a Gradle build script or use the provided quickstart bash script.
+
+### Gradle build script
+
+To execute these build configuration scripts, run the `gradle` command in the directory with these scripts.
+
+**build.gradle**
+
+```gradle
+plugins {
+    id 'java'
+    id 'application'
+    // shadow plugin to produce fat JARs
+    id 'com.github.johnrengelman.shadow' version '7.1.2'
+}
+// artifact properties
+group = 'org.myorg.quickstart'
+version = '0.1-SNAPSHOT'
+mainClassName = 'org.myorg.quickstart.StreamingJob'
+description = """Flink Quickstart Job"""
+ext {
+    javaVersion = '1.8'
+    flinkVersion = '{{< version >}}'
+    scalaBinaryVersion = '{{< scala_version >}}'
+    slf4jVersion = '1.7.32'
+    log4jVersion = '2.17.1'
+}
+sourceCompatibility = javaVersion
+targetCompatibility = javaVersion
+tasks.withType(JavaCompile) {
+	options.encoding = 'UTF-8'
+}
+applicationDefaultJvmArgs = ["-Dlog4j.configurationFile=log4j2.properties"]
+
+// declare where to find the dependencies of your project
+repositories {
+    mavenCentral()
+    maven { url "https://repository.apache.org/content/repositories/snapshots/" }
+}
+// NOTE: We cannot use "compileOnly" or "shadow" configurations since then we could not run code
+// in the IDE or with "gradle run". We also cannot exclude transitive dependencies from the
+// shadowJar yet (see https://github.com/johnrengelman/shadow/issues/159).
+// -> Explicitly define the // libraries we want to be included in the "flinkShadowJar" configuration!
+configurations {
+    flinkShadowJar // dependencies which go into the shadowJar
+    // always exclude these (also from transitive dependencies) since they are provided by Flink
+    flinkShadowJar.exclude group: 'org.apache.flink', module: 'force-shading'
+    flinkShadowJar.exclude group: 'com.google.code.findbugs', module: 'jsr305'
+    flinkShadowJar.exclude group: 'org.slf4j'
+    flinkShadowJar.exclude group: 'org.apache.logging.log4j'
+}
+// declare the dependencies for your production and test code
+dependencies {
+    // --------------------------------------------------------------
+    // Compile-time dependencies that should NOT be part of the
+    // shadow jar and are provided in the lib folder of Flink
+    // --------------------------------------------------------------
+    implementation "org.apache.flink:flink-streaming-java:${flinkVersion}"
+    implementation "org.apache.flink:flink-clients:${flinkVersion}"
```

Comment:
@matriv This is different than what you propose in https://github.com/apache/flink-web/pull/504/files.

Reply:
It's a mistake I did there, I'll fix it. thx!